maps/z2.dmm

```diff
@@ -82478,7 +82478,8 @@
 	},
 /obj/item/plant/flower/rose{
 	burning = 0;
-	desc = "By any other name, would smell just as sweet. This one likes to be called Lily."
+	desc = "By any other name, would smell just as sweet. This one likes to be called Lily. Or ";
```

Comment:
Did you mean to leave the `Or ` unfinished?

Reply:
Yes, that's intended - The rose object adds a name at the end, so the hanging Or ends up looking like:

"This one likes to be called Lily. Or Elizabeth"

Instead of

"This one likes to be called Lily.Elizabeth"

Reply:
That makes sense!